src/main/kotlin/com/markskelton/settings/ThemeSettingsUI.kt

```diff
@@ -0,0 +1,132 @@
+package com.markskelton.settings
+
+import com.intellij.ide.BrowserUtil.browse
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.options.SearchableConfigurable
+import com.intellij.openapi.project.DumbAware
+import com.intellij.openapi.ui.DialogPanel
+import com.intellij.ui.layout.panel
+import com.markskelton.settings.ThemeSettings.Companion.constructSettingModel
+import java.net.URI
+import javax.swing.JComponent
+
+data class ThemeSettingsModel(
+  var isBold: Boolean,
+  var isVivid: Boolean,
+  var isItalic: Boolean
+)
+
+class ThemeSettingsUI : DumbAware, SearchableConfigurable {
+
+  companion object {
+    const val THEME_SETTINGS_DISPLAY_NAME = "One Dark Theme Settings"
```

Comment:
I feel like just `One Dark Theme` would make more sense for the settings sidebar name.  Thoughts?

```suggestion
    const val THEME_SETTINGS_DISPLAY_NAME = "One Dark Theme"
```

Reply:
:thinking: Ah! I just did that to differentiate it from the other One Dark things. 
![image](https://user-images.githubusercontent.com/15972415/82845819-9a16c080-9eab-11ea-892f-c3da7a6093ec.png)

However, I think I am a power user and most people just use the settings menu.